.rubocop.yml

```diff
@@ -3,7 +3,10 @@ AllCops:
   DisplayCopNames: true
 
 Metrics/LineLength:
-  Max: 140
+  Max: 150
```

Comment:
Too long of lines make readability challenging.  Can you please keep it at the previous threshold and adjust your code?

Reply:
@jcoyne - done.